README.md

```diff
@@ -1 +1,43 @@
-# contributor-website-v2
\ No newline at end of file
+# contributor-website-v2
+Contributors website is a one-stop-shop for anyone associated with moja global to get a glimpse of our projects and their working. The website also holds blogs and guides by the moja family to help you better understand our work and hopefully learn from others experience.
```

Comment:
`one-stop-shop` doesn't seem to very appropriate, in my opinion. Instead of guides, you can mention modules.

Reply:
Sure.